pkg/managerclient/client.go

```diff
@@ -406,11 +406,13 @@ func (c *Client) uniqueTaskID(ctx context.Context, clusterID, taskType string) (
 }
 
 func (c *Client) taskByName(ctx context.Context, clusterID, taskType, taskName string) (uuid.UUID, error) {
+	all := true
 	resp, err := c.operations.GetClusterClusterIDTasks(&operations.GetClusterClusterIDTasksParams{
 		Context:   ctx,
 		ClusterID: clusterID,
 		Type:      &taskType,
 		Short:     pointer.BoolPtr(true),
+		All:       &all,
```

Comment:
Why not use `pointer.BoolPtr(true)` like above?

Reply:
shame on me :) fixing...